### ISSUE TITLE
error using openshift-installer ocp-4.4 , libvirt, must-gather included

### ISSUE THREAD

**Issue by oshoval** [openshift/okd#91]
Trying to run ocp-4.4 install , 3 masters, libvirt
[must-gather](https://easyupload.io/cyigr9)

i am getting
```
DEBUG Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-03-03-110909: 99% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-03-03-110909: 99% complete, waiting on authentication, console, monitoring 
DEBUG Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, monitoring 
DEBUG Still waiting for the cluster to initialize: Cluster operator machine-config is reporting a failure: Failed to resync 4.4.0-0.nightly-2020-03-03-110909 because: timed out waiting for the condition during syncRequiredMachineConfigPools: error pool master is not ready, retrying. Status: (pool degraded: false total: 3, ready 2, updated: 3, unavailable: 1) 
DEBUG Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-03-03-110909: 99% complete 
DEBUG Still waiting for the cluster to initialize: Cluster operator machine-config is reporting a failure: Failed to resync 4.4.0-0.nightly-2020-03-03-110909 because: timed out waiting for the condition during syncRequiredMachineConfigPools: error pool master is not ready, retrying. Status: (pool degraded: false total: 3, ready 2, updated: 3, unavailable: 1) 
```

looked a bit in the journalctl of that node, looks like it gets timeouts communicating with etcd, hence its kubelet cant report it status and it goes not ready

```
[root@7f84d2725e5e /]# oc adm must-gather
[must-gather      ] OUT Using must-gather plugin-in image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fa7be657bf10ab4f26e5863bf9872f9af364cd09a7a09156529b50230c597865
[must-gather      ] OUT namespace/openshift-must-gather-rxwhl created
[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-vzmmf created
[must-gather      ] OUT pod for plug-in image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:fa7be657bf10ab4f26e5863bf9872f9af364cd09a7a09156529b50230c597865 created
[must-gather-zl8st] OUT gather logs unavailable: Get https://192.168.126.51:10250/containerLogs/openshift-must-gather-rxwhl/must-gather-zl8st/gather?follow=true: x509: certificate signed by unknown authority
[must-gather-zl8st] OUT waiting for gather to complete
[must-gather-zl8st] OUT downloading gather output
WARNING: rsync command not found in path. Please use your package manager to install it.
[must-gather-zl8st] OUT ./version
[must-gather-zl8st] OUT ./cluster-scoped-resources/config.openshift.io/clusterversions/version.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-4577f.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-4m47t.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-7s92q.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-9ntfs.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-mmcx4.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-php46.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-r9dq5.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-sg72n.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-t5hxj.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/certificates.k8s.io/certificatesigningrequests/csr-v47pk.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/machineconfiguration.openshift.io/machineconfigpools/master.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/machineconfiguration.openshift.io/machineconfigpools/worker.yaml
[must-gather-zl8st] OUT ./cluster-scoped-resources/operator.openshift.io/networks/cluster.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/openshift-cluster-version.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-2bbzr/cluster-version-operator-659488f88f-2bbzr.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-2bbzr/cluster-version-operator/cluster-version-operator/logs/previous.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-2bbzr/cluster-version-operator/cluster-version-operator/logs/current.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-2bbzr/cluster-version-operator/cluster-version-operator/logs/previous.insecure.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-v589c/cluster-version-operator-659488f88f-v589c.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-v589c/cluster-version-operator/cluster-version-operator/logs/previous.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-v589c/cluster-version-operator/cluster-version-operator/logs/current.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-v589c/cluster-version-operator/cluster-version-operator/logs/previous.insecure.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/pods/cluster-version-operator-659488f88f-v589c/cluster-version-operator/cluster-version-operator/logs/current.insecure.log
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/pods.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/replicationcontrollers.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/services.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/configmaps.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/events.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/endpoints.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/persistentvolumeclaims.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/core/secrets.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/apps/daemonsets.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/apps/deployments.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/apps/replicasets.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/apps/statefulsets.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/autoscaling/horizontalpodautoscalers.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/batch/jobs.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/batch/cronjobs.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/apps.openshift.io/deploymentconfigs.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/build.openshift.io/buildconfigs.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/build.openshift.io/builds.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/image.openshift.io/imagestreams.yaml
[must-gather-zl8st] OUT ./namespaces/openshift-cluster-version/route.openshift.io/routes.yaml
[must-gather-zl8st] OUT ./namespaces/default/default.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/pods.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/replicationcontrollers.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/services.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/configmaps.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/events.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/endpoints.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/persistentvolumeclaims.yaml
[must-gather-zl8st] OUT ./namespaces/default/core/secrets.yaml
[must-gather-zl8st] OUT ./namespaces/default/apps/daemonsets.yaml
[must-gather-zl8st] OUT ./namespaces/default/apps/deployments.yaml
[must-gather-zl8st] OUT ./namespaces/default/apps/replicasets.yaml
[must-gather-zl8st] OUT ./namespaces/default/apps/statefulsets.yaml
[must-gather-zl8st] OUT ./namespaces/default/autoscaling/horizontalpodautoscalers.yaml
[must-gather-zl8st] OUT ./namespaces/default/batch/jobs.yaml
[must-gather-zl8st] OUT ./namespaces/default/batch/cronjobs.yaml
[must-gather-zl8st] OUT ./namespaces/default/apps.openshift.io/deploymentconfigs.yaml
[must-gather-zl8st] OUT ./namespaces/default/build.openshift.io/buildconfigs.yaml
[must-gather-zl8st] OUT ./namespaces/default/build.openshift.io/builds.yaml
[must-gather-zl8st] OUT ./namespaces/default/image.openshift.io/imagestreams.yaml
[must-gather-zl8st] OUT ./namespaces/default/route.openshift.io/routes.yaml
[must-gather-zl8st] OUT ./audit_logs/openshift-apiserver/test-1-7jskc-master-1-audit.log.gz
[must-gather-zl8st] OUT ./audit_logs/openshift-apiserver/test-1-7jskc-master-2-audit.log.gz
[must-gather-zl8st] OUT ./audit_logs/openshift-apiserver.audit_logs_listing
[must-gather-zl8st] OUT ./audit_logs/kube-apiserver.audit_logs_listing
[must-gather-zl8st] OUT ./host_service_logs/masters/kubelet_service.log
[must-gather-zl8st] OUT ./host_service_logs/masters/crio_service.log
[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-vzmmf deleted
[must-gather      ] OUT namespace/openshift-must-gather-rxwhl deleted
[root@7f84d2725e5e /]# 
```

thanks

**Comment by oshoval** [comment]
as a WA - looks like running systemctl restart kubelet on that node made the master-0 Ready again, 
letting it run more to see if the install will finish

**Comment by sgreene570** [comment]
@oshoval  Are you trying to install OCP 4.4 or OKD 4.4? If OKD, which OKD release did you install from?

**Comment by oshoval** [comment]
@sgreene570 
hi, using OCP-4.4 using :release-4.4 head
since then i managed to install several times, 
in some cases the installer hangs and then i reset the node / kubelet and it continue fine,
also after install i encounter sometimes that if i shutdown all machines and then restart them (this is how we are using clusters that are pre installed one time, and reusable , in a container image),
sometimes some nodes are NotReady, again a restart on them fixes it, 
i can either open other issue on this, or just add here logs when it happens if you want 

thanks

**Comment by vrutkovs** [comment]
No master logs in must-gather

**Comment by oshoval** [comment]
lets close this issue please
in case there is a guide on how to gather logs before and also after installer finishes it would be great please, and then i can open new issue when i have new logs

thanks
